registry/tests/templates.spec.ts

```diff
@@ -576,6 +576,57 @@ describe(`Tests ${example.url}`, () => {
         });
     });
 
+    describe('Partial Update', () => {
+        it("should not partial update any record if record doesn't exist", async () => {
```

Comment:
(minor) should not partial update > should not partially update 
not so robotic